wof/WaterML.py

```diff
@@ -1,4 +1,5 @@
 #!/usr/bin/env python
+
```

Comment:
Sorry for the noise. I made a global search and replace for shebangs and even the "good ones" got an extra line in the change.